svthalia/concrexit
# Generate model relation diagram in CI

Closes #2840



### Summary
Requires [GraphViz](https://graphviz.org/download/)
Create workflow that generates a graph png file (could be dot) of the selected application models (all for now).
For configuration see: [django_extensions graph_model](https://django-extensions.readthedocs.io/en/latest/graph_models.html#default-settings)

### How to test
1. Create a PR
2. See a correct model in models_graph.png

For the commit that fixed the poetry.lock conflict:
[Actions run](https://github.com/svthalia/concrexit/actions/runs/4417743204)
